app.js

```diff
@@ -21,10 +23,16 @@ app.use(logger('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cleanStrings);
-
+app.use(session({
+  secret: 's3cr3t',
```

Comment:
This should be in the environment variables

Reply:
This is true. I'll implement this ASAP